spec/features/choose_lead_image_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+RSpec.feature "Choose a lead image" do
+  scenario "User chooses an existing image as lead image" do
+    given_there_is_a_document_with_existing_images
+    when_i_visit_the_summary_page
+    when_i_visit_the_lead_images_page
+    then_i_should_be_able_to_see_a_list_of_existing_images
+    when_i_remove_the_lead_image
+    then_i_should_not_see_a_lead_image_on_the_summary_page
+    when_i_visit_the_lead_images_page
+    and_i_select_an_image_to_be_the_lead_image
+    then_i_should_see_the_new_lead_image_on_the_summary_page
+  end
+
+  def given_there_is_a_document_with_existing_images
+    document_type_schema = build(:document_type_schema, lead_image: true)
+    document = create(:document, document_type: document_type_schema.id)
+    @image1 = create(:image, document: document, filename: "image-1.jpg",
+                              alt_text: "image 1 alt text", caption: "image 1 caption",
```

Comment:
Could the factory do this for us?

Reply:
Metadata isn't required for an image so I'd rather pass in the values when they're needed than set them as a default in the factory.

Reply:
Ah, yes, good point!